scripts/script.js

```diff
@@ -0,0 +1,77 @@
+var answers = [];
+
+
+
+// questions constructor
+
+var Questions = function(flavor, question){
+	this.question = question;
+	this.flavor = flavor;
+}
+
+
+//  question bank
+
+var questions = [
+
+	new Questions("strong", "Do ye like yer drinks strong?"), 
+	new Questions("salty", "Do ye like it with a salty tang?"), 
+	new Questions("fruity", "Are ye one for a fruity finish?"), 
+	new Questions("sweet", "Would ye like a bit of sweetness with yer poison?")
+]
+
+
+
+// Ingredient contructor function
+
+var Ingredient = function(flavor, ingredient){
+	this.ingredient = ingredient;
+	this.flavor = flavor;
+}
+
+
+// pantry bank
+
+var pantry = [
+	
+	new Ingredient("strong", "Glug of rum, slug of whisky, splash of gin"), 
+	new Ingredient("salty", "Olive on a stick, salt-dusted rim, rasher of bacon"), 
+	new Ingredient("fruity", "Slice of orange, dash of cassis, cherry on top"), 
+	new Ingredient("sweet", "Sugar cube, spoonful of honey, splash of cola")
+
+]
+
+
+var addToList = function(e){
```

Comment:
It works better in an OOP approach that all of your application have been building over Objects so, why don't create a Pantry object and a List object with an add method each one?